src/sentry/integrations/opsgenie/utils.py

```diff
@@ -76,15 +76,11 @@ def send_incident_alert_notification(
         logger.info("Opsgenie team removed, but the rule is still active.")
         return False
 
-    integration_key = team["integration_key"]
-
-    # TODO(hybridcloud) Use integration.get_keyring_client instead.
-    client = OpsgenieClient(
-        integration=integration,
-        integration_key=integration_key,
-        org_integration_id=incident.organization_id,
```

Comment:
This was what caused the bug though 🕵️

Reply:
What was the bug here?  This implementation looks really close to the keyring client init code 🤔

Reply:
The bug is that `organization_id` should not be be used for the organization integration id.